staging/src/k8s.io/kubectl/pkg/cmd/auth/auth.go

```diff
@@ -34,6 +34,7 @@ func NewCmdAuth(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	}
 
 	cmds.AddCommand(NewCmdCanI(f, streams))
+	cmds.AddCommand(NewCmdWhoAmI(f, streams))
```

Comment:
You can't add a GA command, EVERY new command has to be initially created under `kubectl alpha` and marked as such.

Reply:
Oh, sorry. Completly overlooked it, my bad. Fixed.